### PR TITLE
Revised Nomad Enterprise page

### PR DESCRIPTION
**Pull request by yishan-lin**
- Revised Enterprise content to reflect new pricing & packaging modules
- Consolidated feature pages into single Enterprise pager
- Made each feature into its own subsection on the Enterprise page
- Added direct links to their respective guides below
- Upleveled all Enterprise features onto the navbar
- Updated redirects for the now-removed pages

### CODE REVIEW

File: website/source/docs/enterprise/index.html.md [hunk 1]
```diff
@@ -9,24 +9,57 @@ description: |-
...
+Enable multiple teams to safely use a shared multi-region deployment and reduce cluster fleet size.  In Nomad Enterprise, a shared cluster can be partitioned into namespaces which allows jobs and their associated objects to be isolated from each other and other users of the cluster.
 
+Namespaces enhance the usability of a shared cluster by isolating teams from the jobs of others, provide fine grain access control to jobs when coupled with ACLs, and can prevent bad actors from negatively impacting the whole cluster.
+
+See the [Namespaces Guide](https://www.nomadproject.io/docs/enterprise/namespaces/index.html) for a thorough overview.
+
+### Resource Quotas
+Limit resource consumption across teams or projects to reduce waste and align budgets.  In Nomad Enterprise, operators can define quota specifications and apply them to namespaces. When a quota is attached to a namespace, the jobs within the namespace may not consume more resources than the quota specification allows.
+
+This allows operators to partition a shared cluster and ensure that no single actor can consume the whole resources of the cluster. See the Resource Quotas Guide for more details.
```
Comment (preetapan) [inline website/source/docs/enterprise/index.html.md]: How about something like below instead? 
>This allows operators to partition a shared cluster amongst different teams or projects, and ensure distribution of cluster resources according to business priority.

Comment (yishan-lin) [reply website/source/docs/enterprise/index.html.md]: @preetapan Take a look at the latest commit, should address these directly!

File: website/source/docs/enterprise/index.html.md [hunk 2]
```diff
@@ -9,24 +9,55 @@ description: |-
...
-
-Click [here](https://www.hashicorp.com/go/nomad-enterprise) to set up a demo or request a trial 
-of Nomad Enterprise.
+Nomad Enterprise adds collaboration, operational, and governance capabilities to Nomad.  Nomad Enterprise is available as a base Platform package with an optional Governance & Policy add-on module.
+
+Please navigate the sub-sections for more information about each package and its features in detail.
+
+## Nomad Enterprise Platform
+Nomad Enterprise Platform enables operators to easily upgrade Nomad as well as enhanced performance and availability through Advanced Autopilot features such as Automated Upgrades, Enhanced Read Scalability, and Redundancy Zones.
+
```
Comment (rcgenova) [inline website/source/docs/enterprise/index.html.md]: s/enhanced/enhance

File: website/source/docs/enterprise/index.html.md [hunk 3]
```diff
@@ -9,24 +9,55 @@ description: |-
...
+### Enhanced Read Scalability
+This feature enables an operator to introduce non-voting server nodes to a Nomad cluster. Non-voting servers will receive the replication stream but will not take part in quorum (required by the leader before log entries can be committed). Adding explicit non-voters will scale reads and scheduling without impacting write latency.
+
+See the [Autopilot - Read Scalability](https://www.nomadproject.io/guides/operations/autopilot.html#server-read-and-scheduling-scaling) documentation for a thorough overview.
+
+### Redundancy Zones
+Redundancy Zones enables an operator to deploy a non-voting server as a hot standby server on a per availability zone basis. For example, in an environment with three availability zones one can run one voter and one non-voter in each availability zone, for a total of six servers. If an availability zone is completely lost, only one voter will be lost, so the cluster remains available. If a voter is lost in an availability zone, Nomad will promote the non-voter to voter automatically, putting the hot standby server into service quickly.
+
```
Comment (rcgenova) [inline website/source/docs/enterprise/index.html.md]: recommend changing "one can run" to "an operator can run"

Comment (rcgenova) [inline website/source/docs/enterprise/index.html.md]: "Nomad will promote the non-voter to voter" -> "Nomad will promote the non-voter to a voter"

File: website/source/docs/enterprise/index.html.md [hunk 4]
```diff
@@ -9,24 +9,55 @@ description: |-
...
+## Governance & Policy
+Governance & Policy features are part of an add-on module that enables an organization to securely operate Nomad at scale across multiple teams through features such as Namespaces, Resource Quotas, Sentinel Policies, and Preemption.
 
+### Namespaces
+Namespaces enable multiple teams to safely use a shared multi-region Nomad environment and reduce cluster fleet size. In Nomad Enterprise, a shared cluster can be partitioned into multiple namespaces which allow jobs and their associated objects to be isolated from each other and other users of the cluster.
 
+Namespaces enhance the usability of a shared cluster by isolating teams from the jobs of others, provide fine grain access control to jobs when coupled with ACLs, and can prevent bad actors from negatively impacting the whole cluster.
+
+See the [Namespaces Guide](https://www.nomadproject.io/docs/enterprise/namespaces/index.html) for a thorough overview.
```
Comment (rcgenova) [inline website/source/docs/enterprise/index.html.md]: "provide" -> "by providing"; "can prevent" -> "by preventing"

**Pull request merged**